### PR TITLE
Fix/cross check messages

### PR DESCRIPTION
**Pull request by VladLutskevich**

### CODE REVIEW

File: client/src/components/CrossCheckComments.tsx [hunk 1]
```diff
@@ -21,24 +25,43 @@ export function CrossCheckComments(props: Props) {
...
+              <Timeline>
+                {reviewComment.comments
+                  .slice(0)
+                  .reverse()
+                  .map(({ score, comment, dateTime }, j) => (
+                    <>
+                      <br></br>
+                      <Timeline.Item
+                        key={j}
+                        color={j === 0 ? 'green' : 'gray'}
```
Comment (igorlyatskiy) [inline client/src/components/CrossCheckComments.tsx:47]: remove it into getColor

File: client/src/components/CrossCheckComments.tsx [hunk 2]
```diff
@@ -21,24 +25,43 @@ export function CrossCheckComments(props: Props) {
...
+                        dot={<ClockCircleOutlined style={{ fontSize: '16px' }} />}
+                      >
+                        <div>{formatDateTime(dateTime)}</div>
+                        {score ? (
+                          <p style={{ opacity: j === 0 ? '1' : '0.6' }}>
```
Comment (igorlyatskiy) [inline client/src/components/CrossCheckComments.tsx:52]: Remove into getOpacity 
`const getOpacity = (j) => return j === 0 ? '1' : '0.6'`

File: client/src/components/CrossCheckComments.tsx [hunk 3]
```diff
@@ -21,24 +25,43 @@ export function CrossCheckComments(props: Props) {
...
+                            <b>Score: {score}</b>
+                          </p>
+                        ) : null}
+                        {comment.split('\n').map((text, k) => (
+                          <p key={k} style={{ opacity: j === 0 ? '1' : '0.6' }}>
```
Comment (igorlyatskiy) [inline client/src/components/CrossCheckComments.tsx:57]: same

File: client/src/components/CrossCheckComments.tsx [hunk 4]
```diff
@@ -21,24 +25,43 @@ export function CrossCheckComments(props: Props) {
   const style = { margin: 16, fontStyle: 'italic' };
   return (
     <Col>
-      {comments.map(({ comment, author, score }, i) => (
+      {comments.map((reviewComment, i) => (
```
Comment (igorlyatskiy) [inline client/src/components/CrossCheckComments.tsx:28]: Naming, i is not good

File: client/src/components/CrossCheckComments.tsx [hunk 5]
```diff
@@ -21,24 +25,43 @@ export function CrossCheckComments(props: Props) {
...
-                ) : null}
-                {comment.split('\n').map((text, k) => (
-                  <p key={k}>{text}</p>
-                ))}
-              </>
+              <Timeline>
+                {reviewComment.comments
+                  .slice(0)
+                  .reverse()
+                  .map(({ score, comment, dateTime }, j) => (
```
Comment (igorlyatskiy) [inline client/src/components/CrossCheckComments.tsx:42]: j? Naming

**Review by igorlyatskiy — APPROVED** [review]

**Review by igorlyatskiy — CHANGES_REQUESTED** [review]
Фиксани замечания до пятницы

File: client/src/components/CrossCheckComments.tsx [hunk 6]
```diff
@@ -19,26 +23,51 @@ export function CrossCheckComments(props: Props) {
     return null;
   }
   const style = { margin: 16, fontStyle: 'italic' };
+  const getOpacity = (j: number) => (j === 0 ? '1' : '0.6');
+  const getColor = (j: number) => (j === 0 ? 'green' : 'gray');
```
Comment (igorlyatskiy) [inline client/src/components/CrossCheckComments.tsx:27]: Fix naming

Comment (igorlyatskiy) [reply client/src/components/CrossCheckComments.tsx:27]: commentIndex, not j

**Review by igorlyatskiy — CHANGES_REQUESTED** [review]

**Review by igorlyatskiy — APPROVED** [review]

**Review by igorlyatskiy — APPROVED** [review]

File: client/src/components/CrossCheckComments.tsx [hunk 7]
```diff
@@ -1,11 +1,15 @@
...
+import { formatDateTime } from 'services/formatter';
 
 type Props = {
   comments: {
-    comment: string;
-    score: number;
+    comments: {
+      comment: string;
+      score: number;
+      dateTime: number;
```
Comment (KostyaDedulya) [inline client/src/components/CrossCheckComments.tsx:11]: this can be separate type

File: client/src/components/CrossCheckComments.tsx [hunk 8]
```diff
@@ -19,26 +23,51 @@ export function CrossCheckComments(props: Props) {
     return null;
   }
   const style = { margin: 16, fontStyle: 'italic' };
+  const getOpacity = (commentIndex: number) => (commentIndex === 0 ? '1' : '0.6');
```
Comment (KostyaDedulya) [inline client/src/components/CrossCheckComments.tsx:26]: magic numbers

File: client/src/components/CrossCheckComments.tsx [hunk 9]
```diff
@@ -19,26 +23,51 @@ export function CrossCheckComments(props: Props) {
...
-                  <p key={k}>{text}</p>
-                ))}
-              </>
+              <Timeline>
+                {reviewInfo.comments
+                  .slice(0)
+                  .reverse()
+                  .map(({ score, comment, dateTime }, commentIndex) => (
+                    <>
+                      <br></br>
```
Comment (KostyaDedulya) [inline client/src/components/CrossCheckComments.tsx:50]: <br/>

**Review by KostyaDedulya — CHANGES_REQUESTED** [review]

**Review by igorlyatskiy — APPROVED** [review]

**Pull request merged**